app/frontend/components/shared/permit-applications/requirement-form.tsx

```diff
@@ -28,7 +28,6 @@ import { IErrorsBoxData } from "../../../types/types"
 import { getCompletedBlocksFromForm } from "../../../utils/formio-component-traversal"
```

Comment:
Not sure where the code is but something weird happening here with the auto-compliance note - duplicated text and popover position seems a bit odd (should be at the start of the field?)

<img width="962" alt="Screen Shot 2024-03-25 at 11 26 35 AM" src="https://github.com/bcgov/HOUS-permit-portal/assets/15823290/4ef332ff-8c32-4912-b009-1576d6b77a52">

Reply:
that double stacked happened because I think you need to force refresh the page.
But yeah I've moved the bubble over to the left now instead of centered.